### PR TITLE
Update tutorial to include building on CentOS 8 Stream

### PR DESCRIPTION
**Pull request by fyshhh**
This pull request updates the tutorial to include steps for building MANA on CentOS 8 Stream.

### CODE REVIEW

File: doc/mana-centos-tutorial.txt [hunk 1]
```diff
@@ -2,59 +2,68 @@ This document outlines the necessary steps to take to install and compile MANA
...
 existing packages on your machine.
 
-  $ yum install -y git
-  $ yum groupinstall 'Development Tools'
-  $ yum install -y mlocate vim wget
-  $ yum install -y centos-release-scl
-  $ yum install -y devtoolset-8-gcc devtoolset-8-gcc-c++ glibc-static.x86_64
-  $ echo 'source scl_source enable devtoolset-8' >> ~/.bashrc
-  $ source ~/.bashrc
+### CENTOS 7 ###
```
Comment (JainTwinkle) [inline doc/mana-centos-tutorial.txt:12]: Let's use `$` as the prefix for all the commands and `#` for heading.

File: doc/mana-centos-tutorial.txt [hunk 2]
```diff
@@ -191,18 +200,84 @@ NOTE: In some cases, where there's an Nvidia GPU is installed, one may need to
   install extra libraries (e.g., libX11, libXau, libXext, libxcb) similarly
   as mentioned above.
 
+### CENTOS 8 STREAM ###
+
+These steps are largely similar to the ones above. The SCL repository no longer
+exists on CentOS 8 Stream, and instead the relevant tools are included in
+Development Tools.
+
+  # dnf install -y git
```
Comment (JainTwinkle) [inline doc/mana-centos-tutorial.txt:209]: Let's make # for heading and $ for commands.

Comment (fyshhh) [reply doc/mana-centos-tutorial.txt:209]: sure.

**Review by JainTwinkle — APPROVED** [review]
LGTM!

**Pull request merged**